protocol/interop-monitoring.md

```diff
@@ -0,0 +1,140 @@
+# Interop Monitoring Service
+
+|                    |                                                    |
+| ------------------ | -------------------------------------------------- |
+| Author             | _Mark Tyneway, Axel Kingsley_                                     |
+| Created at         | _2025-03-19_                                       |
+
+## Purpose
+
+This document is meant to align on a strategy for monitoring interop and propose a
+Monitoring Service for Executing Messages.
+
+## Summary + Problem Statement + Context
+
+Given assumptions in the [cloud topology](https://github.com/ethereum-optimism/design-docs/pull/218),
+it is generally not possible to guarantee that invalid `ExecutingMessage`s do not finalize without multiple
+implementations of `op-supervisor`. With only a single implementation, a bug becomes consensus.
+In the worst case, this can mint an infinite amount of ether. Given this risk, we need a monitoring,
+alerting and runbook for handling invalid `ExecutingMessage`s being included in the chain.
```

Comment:
```suggestion
In the worst case, this can mint an infinite amount of ether. Given this risk, we need to have monitoring,
alerting, and a runbook for handling invalid `ExecutingMessage`s being included in the chain.
```